docs/Learning-Environment-Design-Agents.md

```diff
@@ -223,10 +243,22 @@ as observations directly, this is done automatically by the Agent.
 
 ![Agent RenderTexture Debug](images/gridworld.png)
 
+#### Visual Observation Summary & Best Practices
+
+* To collect visual observations, attach `CameraSensor` or `RenderTextureSensor`
+  components to the agent GameObject.
+* Visual observations should only be used when there is no parsimonious set of
+  vector observations to capture the relevant information necessary for an agent to make informed decisions.
```

Comment:
```suggestion
* Visual observations should generally be used unless vector observations are not sufficient
   capture the relevant information necessary for an agent to make informed decisions.
```
Can you de-obfuscate this a bit (unless "parsimonious" is a widely used term inRL literature)? One possible wording proposed.

Reply:
Not an RL term. Just my own jargon. Removed it.